### DISCUSSION TITLE
Looking for clarification on this firmwares possible use with Ender3 s1 pro.

### DISCUSSION THREAD

**Discussion by wtstreetglow** [mriscoc/Ender3V2S1#318] [General]
Clarification. Will this work for the Pro variety with the touchscreen? Thank you ahead of time. Are there any modifications I can make to the Pro to get this to work?-- I.e. switch screen to rotary variety (And still keep the 300 degree Mac temp) End goal full support of octoprint.

Thank you all for the discussion ahead of time!$

**Comment by mriscoc** [comment]
Current Requirements:
https://github.com/mriscoc/Ender3V2S1/wiki#requirements

**Comment by wtstreetglow** [comment]
Thank you for this. I see what I need to do now to get it working!
